libs/lensfun/modifier.cpp

```diff
@@ -162,6 +163,9 @@ int lfModifier::EnableScaling (float scale)
         if (scale == 0.0)
             return false;
     }
+    
+    // backward compatibility to 0.3.2
+    scale /= scaling_d;
```

Comment:
Watch out that `scaling_d` is not yet set to the correct value when `EnableScaling` is called before `EnableDistortion`